src/utils/index.test.js

```diff
@@ -1,5 +1,83 @@
-const { isMaxGuesses } = require('./utils');
+const { EvaluationColor, getEvaluation, isMaxGuesses, isWin } = require('../utils');
 
-test(`test test`, () => {
-    expect(isMaxGuesses({})).toBe(false);
+/*
+ * getEvaluation
+ */
+
+test('evaluations all match', () => {
+    const word = "abcde";
+    const guess = "abcde";
+    const expected = [EvaluationColor.Green, EvaluationColor.Green, EvaluationColor.Green, EvaluationColor.Green, EvaluationColor.Green];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('no evaluations match', () => {
+    const word = "abcde";
+    const guess = "fffff";
+    const expected = [EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one match', () => {
+    const word = "abcde";
+    const guess = "aaaaa";
+    const expected = [EvaluationColor.Green, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one partial', () => {
+    const word = "abcde";
+    const guess = "faaaa";
+    const expected = [EvaluationColor.Gray, EvaluationColor.Yellow, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one match, one partial', () => {
+    const word = "abbde";
+    const guess = "aaaba";
+    const expected = [EvaluationColor.Green, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Yellow, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+test('one match, no yellow overwrite', () => {
+    const word = "abbbb";
+    const guess = "aaccc";
+    const expected = [EvaluationColor.Green, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray, EvaluationColor.Gray];
+    expect(getEvaluation(word, guess)).toEqual(expected);
+})
+
+/*
+ * isMaxGuesses
+ */
+
+test(`not at max guesses`, () => {
+    expect(isMaxGuesses([], 6)).toEqual(false);
+})
+
+test(`at max guesses`, () => {
+    expect(isMaxGuesses([[], [], [], [], [], []], 6)).toEqual(true);
+})
+
+test(`beyond max guesses`, () => {
+    expect(isMaxGuesses([[], [], [], [], [], [], []], 6)).toEqual(true);
+})
+
+/*
+ * isWin
+ */
+
+test(`win`, () => {
```

Comment:
I appreciate the brevity of your titles _but_ it wouldn't hurt to be a little more explicit with what the test is actually testing.  E.g., "a fully green evaluation is a winner" or something.